assets/src/edit-story/components/form/dateTime/datePicker.js

```diff
@@ -39,12 +44,84 @@ function DatePicker({ currentDate, onChange, onViewChange }) {
     [value, onChange]
   );
 
+  const updateTabIndexes = useCallback(
+    (setFocus = false) => {
+      // Set tabIndex to -1 for every except for the first button.
+      if (nodeRef.current) {
+        // Allow tabbing to sections inside the calendar.
+        const navButtons = [
+          ...nodeRef.current.querySelectorAll(
+            '.react-calendar__navigation button'
+          ),
+        ];
+        navButtons[0].tabIndex = '0';
+        navButtons.shift();
+        for (const btn of navButtons) {
+          btn.tabIndex = '-1';
+        }
+
+        // Dates / days.
+        const buttons = [
+          ...nodeRef.current.querySelectorAll(
+            '.react-calendar__viewContainer button'
+          ),
+        ];
+
+        let foundActive = false;
+        for (const btn of buttons) {
+          if (!btn.className.includes('react-calendar__tile--now')) {
```

Comment:
You can use [`.classList.contains()`](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/contains) here.